kyo-data/shared/src/main/scala/kyo/Chunk.scala

```diff
@@ -30,7 +30,9 @@ sealed abstract class Chunk[A] extends Seq[A] derives CanEqual:
       * @return
       *   true if the Chunk contains no elements, false otherwise
       */
-    final override def isEmpty: Boolean = size == 0
+    final override def isEmpty: Boolean = length == 0
```

Comment:
Do you want this to be private/protected?

Reply:
It's not possible because it overrides the super class method